### DISCUSSION TITLE
Binding value not written to table

### DISCUSSION THREAD

**Discussion by JeroenP1974** [Budibase/budibase#13021] [Help] [answered]
Hello everybody, 

I have a screen with a form that can be used to add company information. The form also contains a field that is a relationship with another table that contains personal data of the owner of the company. The value (owner name) of this field is automatically taken from a form on a previous screen. Everything is working. The right value is visible, but when I want to save the data with an action button, the data of this field is not written to the (internal budibase) table. When I physically clicked the data in the form then the data wil be written. But I prefer that this will be automatically field and written. How do I get this working? Is there a solution? All other data in the form is saved. 

Thank you for answers. 

<img width="925" alt="Scherm­afbeelding 2024-02-13 om 00 39 04" src="https://github.com/Budibase/budibase/assets/157612513/8c9027db-1f2b-4faa-b04d-e4ce134ed4ae">

<img width="1064" alt="Scherm­afbeelding 2024-02-13 om 00 41 36" src="https://github.com/Budibase/budibase/assets/157612513/5835393f-5470-4fc5-818e-e82a5c439be9">

**Comment by ConorWebb96** [comment, marked as answer]
Hey @JeroenP1974,

This seems to have been indirectly fixed by one of our releases. Can you update to the most recently release and let us know if this is working now for you.
![20240216_085813](https://github.com/Budibase/budibase/assets/126772285/f9389194-dd92-4869-9112-a83098a844b0)
